src/App.tsx

```diff
@@ -8,37 +8,142 @@ import { PostsList } from './components/PostsList';
 import { PostDetails } from './components/PostDetails';
 import { UserSelector } from './components/UserSelector';
 import { Loader } from './components/Loader';
+import { User } from './types/User';
+import { getPostComments, getUserPosts, getUsers } from './api';
+import { Post } from './types/Post';
+import { Comment } from './types/Comment';
+import { Error } from './types/Error';
 
 export const App: React.FC = () => {
+  const [users, setUsers] = useState<User[]>([]);
+  const [isLoading, setIsLoading] = useState('');
```

Comment:
Pls make `isLoading` boolean and use it in in your code as boolean so that its mission is clear
And for the future - all consts/variables starting with `is` should be boolean as they are basically answering Yes/No question. For example, here you kind of asking _is this loading?_ so the answer is either _yes it is_ or _no it isn't_ these could be represented by true or false